### ISSUE TITLE
Independent builds per Page / Component

### ISSUE THREAD

**Issue by JulianIsrael** [iansinnott/react-static-webpack-plugin#29]
Hi, is there a way to implement independent bundles for a page, without rebuilding the entire project and just serving the new bundle ? with that would be possible to have independent build process for pages.

**Comment by iansinnott** [comment]
Do you mean code splitting https://webpack.github.io/docs/code-splitting.html? If so then you should be able to do it with webpack directly. But I may be misunderstanding your use case. If so, could you elaborate a bit?

**Comment by JulianIsrael** [comment]
Right, what I mean is having a very large application where you could use build versions for each page / modules / blades and whenever only one of these modules requires a change don't requiring to build the entire app, so you would be able to do a build for that particular page (module) and do a bundle swap to production when that module is ready. Thanks for the quick answer.

**Comment by iansinnott** [comment]
Hm, i'm still not sure I understand this use case as opposed to code splitting. Is there another build system that does what you're talking about?

**Comment by JulianIsrael** [comment]
Thanks, maybe webpack code splitting already covers my use case and I haven't seeing it yet. Will take a deeper look.
